server/sonar-web/src/main/js/components/navigator/facets-view.js

```diff
@@ -40,10 +40,10 @@ export default Marionette.CollectionView.extend({
   },
 
   updateState () {
-    const enabledFacets = this.collection.filter(function (model) {
+    const enabledFacets = this.collection.filter(model => {
       return model.get('enabled');
     });
-    const enabledFacetIds = enabledFacets.map(function (model) {
+    const enabledFacetIds = enabledFacets.map(model => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)